docs/tables/scaleway_kubernetes_pool.md

```diff
@@ -0,0 +1,70 @@
+# Table: scaleway_kubernetes_pool
+
+A Scaleway Kubernetes is a public cloud mamanged kubernetes available in Scaleway.
```

Comment:
@jplanckeel could we please update the table description here? It is exactly similar to the one we have used for the kubernetes_cluster table.

Reply:
yes is update ;)